Misc/NEWS.d/next/Library/2024-07-06-12-37-10.gh-issue-121423.vnxrl4.rst

```diff
@@ -0,0 +1,2 @@
+Improve import time of `mod`:`socket` by writing `socket.errorTab` as a 
+constant and lazy import of `mod`:`selectors`
```

Comment:
```suggestion
Improve import time of :mod:`socket` by writing :data:`!socket.errorTab` as a 
constant and lazy import of :mod:`selectors`.
```

Reply:
Rewording. I know it wouldn't have made sense in the first place but I thought that the lazy import actually referred to `socket.errorTab` so with this reformulation (and Sphinx correction) I think it could be clearer.